File: PersonnelManagement/PersonnelManagement/Front/classes/Requests_classes/education_request.ts
import StructureTree from '../../classes/structuretree';

export default class Education_Request {
    constructor() {
        this.current_structure = [];
        this.educationLevel = [];
        this.specializationList = [];
        this.cvalificationList = [];
    };
    current_structure: StructureTree[];
    specializationList: string[];
    cvalificationList: string[];
    educationLevel: string[];
}

File: PersonnelManagement/PersonnelManagement/Models/Education_Request.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace PersonnelManagement.Models
{
    public partial class Education_Request
    {
        public IEnumerable<StructureTree> Current_structure { get; set; } 
        public List<string> SpecializationList { get; set; } 
        public List<string> CvalificationList { get; set; } 
        public List<string> EducationLevel { get; set; } 
    }
}
